src/keepers/AlchemixHarvesterOptimism.sol

```diff
@@ -0,0 +1,57 @@
+pragma solidity ^0.8.13;
+
+import "../../lib/openzeppelin-contracts/contracts/access/Ownable.sol";
+import "./AlchemixGelatoKeeper.sol";
+import "../interfaces/IAlchemistV2.sol";
+import "../interfaces/keepers/IHarvestResolver.sol";
+import "../interfaces/keepers/IAlchemixHarvesterOptimism.sol";
+import "../interfaces/ISidecar.sol";
+
+contract AlchemixHarvesterOptimism is IAlchemixHarvesterOptimism, AlchemixGelatoKeeper {
+  /// @notice The address of the resolver.
+  address public resolver;
+
+  /// @notice The address of the sidecar.
+  address public sidecar;
+
+  constructor(
+    address _gelatoPoker,
+    uint256 _maxGasPrice,
+    address _resolver,
+    address _sidecar
+  ) AlchemixGelatoKeeper(_gelatoPoker, _maxGasPrice) {
+    resolver = _resolver;
+    sidecar = _sidecar;
+  }
+
+  function setResolver(address _resolver) external onlyOwner {
```

Comment:
Line 173 has set sidecar already

Reply:
thats the resolver. this is the harvester.

but on second thought, lets not set the sidecar address in the constructor. it should be passed in to the harvest call since it will be dynamic like the alchemist address